### ISSUE TITLE
Use of `download()` without `library(downloader)` or `downloader`

### ISSUE THREAD

**Issue by lwaldron** [genomicsclass/labs#87]
At: https://genomicsclass.github.io/book/pages/permutation_tests_exercises.html

The line:
```
download(url, destfile=filename)
```

gives an error could not find function "download"; should either be `downloader::download(url, destfile=filename)` or `download.file`

**Comment by vjcitn** [comment]
I can't find the sources for this at https://github.com/genomicsclass/labs/blob/master/inference/permutation_tests.Rmd ...
any hints?

**Comment by mikelove** [comment]
Yeah, I’ve found that the labs repo was cleaned up / reorganized (don’t know by whom) and no longer contains source for all the html pages in the book repo. I’ve found some of my Rmd files missing though the html are still live.

**Comment by mikelove** [comment]
An example: this html file showing the links between PCA and SVD in R was popular on twitter but I found the Rmd was scrubbed from labs. I added a link because someone asked where the link had gone but the source is now gone.

https://github.com/genomicsclass/book/commit/9dab8bc56b2fab07b565cb1755896065e6b69302
